doc/source/whatsnew/v0.20.0.txt

```diff
@@ -29,8 +29,7 @@ New features
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
-
-
+- Strings passed to ``DataFrame.groupby()`` as the ``by`` parameter may now reference either column names or index level names (:issue:`5677`)
```

Comment:
this will need an example here. put the same one in groupby.rst (make also need to add to the groupby doc-string)